salt/cloud/deploy/bootstrap-salt.sh

```diff
@@ -1295,7 +1295,7 @@ __gather_system_info() {
 #----------------------------------------------------------------------------------------------------------------------
 # shellcheck disable=SC2034
 __ubuntu_derivatives_translation() {
-    UBUNTU_DERIVATIVES="(trisquel|linuxmint|linaro|elementary_os|neon)"
```

Comment:
this should stay neon and all changes in this file. we should not update the bootstrap script so remove all changes here. There is an ubuntu derivative named neon